### DISCUSSION TITLE
Which components temperature is displayed?

### DISCUSSION THREAD

**Discussion by pauljurczak** [memtest86plus/memtest86plus#215] [Q&A] [answered]
`CLK/Temp` line shows two different temperatures, e.g. `43/57`. I'm assuming that one of them is of the CPU. What is the other one of? In what order?

**Comment by debrouxl** [comment, marked as answer]
Both temperatures are for the CPU: current/maximum reached during test.

**Reply by pauljurczak** [reply]
Thank you.
